Compentio.SourceMapper.Generator/Extensions/ExtException.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Compentio.SourceMapper.Extensions
```

Comment:
It would be good to implement diagnostics without throwing Exeptions

Reply:
It is only report not throw - diagnostics should be work as we expected.